app/src/main/java/com/swent/assos/ui/components/HomeItem.kt

```diff
@@ -25,21 +29,60 @@ import androidx.compose.ui.text.font.FontFamily
 import androidx.compose.ui.text.font.FontWeight
 import androidx.compose.ui.unit.dp
 import androidx.compose.ui.unit.sp
+import androidx.hilt.navigation.compose.hiltViewModel
 import coil.compose.AsyncImage
 import com.swent.assos.R
 import com.swent.assos.model.data.News
 import com.swent.assos.model.navigation.Destinations
 import com.swent.assos.model.navigation.NavigationActions
+import com.swent.assos.model.view.EventViewModel
+import com.swent.assos.model.view.NewsViewModel
+import java.time.LocalDateTime
 
 @Composable
-fun HomeItem(news: News, navigationActions: NavigationActions) {
+fun HomeItem(id: String, navigationActions: NavigationActions) {
+
+  val viewModel: NewsViewModel = hiltViewModel()
+  val allNews by viewModel.allNewsOfAllAssos.collectAsState()
+
+  val eventViewModel: EventViewModel = hiltViewModel()
+  val event by eventViewModel.event.collectAsState()
+
+  val title: String
+  val description: String
+  var assoId: String = ""
+
+  val news =
+      allNews.find { it.id == id }
+          ?: News(
+              id = "0000",
+              title = "NO TITLE",
+              description = "NO DESCRIPTION",
+              images = listOf(Uri.EMPTY),
+              createdAt = LocalDateTime.now(),
+              associationId = "",
+              eventIds = mutableListOf("0000"))
```

Comment:
Why is eventIds a list if we only take into account the first item of that list ?